.github/workflows/pr.yml

```
@@ -0,0 +1,30 @@
+on: pull_request
+name: Review
+jobs:
+  changelog:
+    runs-on: ubuntu-latest
+    name: Changelog should be updated
+    strategy:
+      fail-fast: false
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+
+      - name: Git fetch
+        run: git fetch
+
+      - name: Check that changelog has been updated.
+        run: git diff --exit-code origin/master -- CHANGELOG.md && exit 1 || exit 0
+  assets-coding-standards:
```

Add a blank line before each action for better readability:

```suggestion

  assets-coding-standards:
```